File: UI/eairflow_desktop/lib/models/airport.dart
class Airport {
  int? airportId;
  String? name;
  String? city;
  String? country;
  String? timeZone;

  Airport({
    this.airportId,
    this.name,
    this.city,
    this.country,
    this.timeZone,
  });

  factory Airport.fromJson(Map<String, dynamic> json) => Airport(
        airportId: json['airportId'] as int?,
        name: json['name'] as String?,
        city: json['city'] as String?,
        country: json['country'] as String?,
        timeZone: json['timeZone'] as String?,
      );

  Map<String, dynamic> toJson() => {
        'airportId': airportId,
        'name': name,
        'city': city,
        'country': country,
        'timeZone': timeZone,
      };
}

File: eAirFlow.Model/Models/Airport.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eAirFlow.Model.Models
{
    public class Airport
    {
        public int AirportId { get; set; }
        public string Name { get; set; }
        public string City { get; set; }
        public string Country { get; set; }
        public string? TimeZoneId { get; set; }

    }
}
